### ISSUE TITLE
Command Line/Web Interface Team Communication

### ISSUE THREAD

**Issue by johnsc1** [GatorCogitate/cogitate_tool#67]
This issue is for communication between the Command Line Interface Team (branch: [command-line-interface](https://github.com/GatorCogitate/cogitate_tool/tree/command-line-interface)) and the Web Interface Team (branch: [WebInterface](https://github.com/GatorCogitate/cogitate_tool/tree/WebInterface)) regarding the integration of the two interfaces.

**Comment by cklima616** [comment]
**Commands we have:**
- link
- token
- repo
- team/individual 

**Levels to Commands:**
1. team/individual
2. params needed -> calling functions -> results
3. link to streamlit

**Comment by lussierc** [comment]
Hi all, to let you know, the Web Interface team is continuing work on implementing a singular function (which will eventually call other functions for graph generation) as requested. We are just waiting on the Feature teams to complete work. 

Once we have implemented functionality for our graphs to work with the soon to be finished features, we will let you know what the main function is for the Web Interface.

**Comment by gkapfham** [comment]
What it the status on this issue?

**Comment by lussierc** [comment]
With all interfaces and features being merged into ‘master’, it seems we no longer need this Issue. @gkapfham
